packages/core/utils/src/dml/__tests__/entity_builder.spec.ts

```diff
@@ -122,6 +122,56 @@ describe("Entity builder", () => {
     ])
   })
 
+  test("define an entity with default value", () => {
+    const model = new EntityBuilder()
+    const user = model.define("user", {
+      id: model.number(),
+      username: model.text().defaultsTo("foo"),
```

Comment:
suggestion: I would prefer `default` instead of `defaultsTo` wdyt?

Reply:
+1

Reply:
Sure. Seems like Oli asked for the same.